gensim/test/test_corpora_dictionary.py

```diff
@@ -359,6 +359,30 @@ def test_patch_with_special_tokens(self):
         self.assertNotIn((1, 1), d.doc2bow(corpus_with_special_tokens[0]))
         self.assertIn((1, 1), d.doc2bow(corpus_with_special_tokens[1]))
 
+    def test_most_common_with_n(self):
+        texts = [['human', 'human', 'human', 'computer', 'computer', 'interface', 'interface']]
+        d = Dictionary(texts)
+        expected = [('human', 3), ('computer', 2)]
+        self.assertEqual(d.most_common(n=2), expected)
```

Comment:
We use pytest, so please use the simpler assertion syntax, here and everywhere.

```suggestion
        assert d.most_common(n=2) == expected
```

Reply:
Forgive my ignorance. Is this really equivalent from the viewpoint of informative error messages?

Reply:
Also: overwhelming weight-of-existing-unit-test-code (in this file and elsewhere) seems to prefer the assertion-methods over the `assert` keyword.

Reply:
> Is this really equivalent from the viewpoint of informative error messages?

Yes, pytest takes care of that.

> overwhelming weight-of-existing-unit-test-code (in this file and elsewhere) seems to prefer the assertion-methods over the assert keyword.

Most tests were written before gensim started using pytest in 2017. The trade-off here is test readability vs consistency with an older test framework (unittest has it's pros and cons, but we've been using pytest for years, so let's not debate them here). My opinion is that for newer tests, readability is more important than consistency.

Reply:
Aha. Well, if keyword-style has some advantages under `pytest` such that it's now preferred style, that could be declared on the project style guidelines, including some hint about if/when older usages should be changed, and whether matching existing style of a method/class or new-style should be preferred.

Reply:
I have added the pytest assertion syntax in 4aacad27.